src/main/kotlin/wordle/domain/Words.kt

```diff
@@ -0,0 +1,65 @@
+package wordle.domain
+
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+class Words(private val values: List<Word>) {
+
+    private val answer: Word = findAnswer()
+    private var answerMap: MutableMap<Char, Int> = HashMap()
+
+    private fun findAnswer(): Word {
+        val date: LocalDate = LocalDate.now()
+        val standardDate: LocalDate = LocalDate.of(2021, 6, 19)
+        val days: Int = ChronoUnit.DAYS.between(standardDate, date).toInt()
+        return values[days % values.size]
+    }
+
+    fun contains(word: Word): Boolean = values.contains(word)
+
+    fun isCorrect(word: Word): Boolean {
+        return word == answer
+    }
+
+    fun check(word: Word): List<Tile> {
+        answerMap = initAnswerMap()
+        val result = MutableList(WORD_SIZE) { Tile.GRAY }
+
+        repeat(WORD_SIZE) { putTileIfSame(result, word, it) }
+        repeat(WORD_SIZE) { putTileIfContains(result, word, it) }
+        return result
+    }
+
+    private fun initAnswerMap(): MutableMap<Char, Int> {
+        val answerMap: MutableMap<Char, Int> = HashMap()
+        answer.value.forEach {
+            answerMap[it] = answerMap.getOrDefault(it, 0) + 1
+        }
+        return answerMap
+    }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private fun initAnswerMap(): Map<Char, Int> {
        return answer.value
            .groupBy { it }
            .mapValues { (_, values) -> values.count() }
    }
```